### PR TITLE
[ADD] runbot_gitlab: for v9.0

### PR DESCRIPTION
**Pull request by moylop260**

### CODE REVIEW

File: runbot_gitlab/models/runbot_build.py [hunk 1]
```diff
@@ -45,12 +45,11 @@ def _compute_dest(self):
 
     @api.multi
     def _prepare_gitlab_status_post(self, state):
-        self.ensure_one()
```
Comment (moylop260) [inline runbot_gitlab/models/runbot_build.py]: Is really required delete it?
I mean, you have a `return`

Comment (JesusZapata) [reply runbot_gitlab/models/runbot_build.py]: @moylop260 
No, finger error
Thanks

File: runbot_send_email/models/runbot_build.py [hunk 2]
```diff
@@ -195,6 +195,8 @@ def action_send_email(self):
     def send_email(self):
         partner_obj = self.env['res.partner']
         for record in self:
```
Comment (moylop260) [inline runbot_send_email/models/runbot_build.py:197]: What about `for record in self.filtered('commiter_email')`?

**Comment by JesusZapata** [comment]
@moylop260 
We close this PR to use the new module runbot_gitlab_minimal https://github.com/Vauxoo/runbot-addons/pull/90?

**Comment by moylop260** [comment]
Superseded by https://github.com/Vauxoo/runbot-addons/pull/90

**Pull request closed without merging**